control-plane/pkg/security/secret.go

```diff
@@ -39,6 +39,8 @@ const (
 	SaslMechanismKey = "sasl.mechanism"
 	SaslUserKey      = "user"
 	SaslPasswordKey  = "password"
+	SaslType         = "saslType" // legacy secrets
```

Comment:
Should we remove them (and release note) in a future release?

Reply:
+1 on that, especially addressing the lack of docs on secret types and auth for channels and brokers on knative.dev